modules/codegen-cli/src/smithy4s/codegen/cli/DumpModel.scala

```diff
@@ -16,7 +16,7 @@
 
 package smithy4s.codegen.cli
 
-import smithy4s.codegen.ModelLoader
+import smithy4s.codegen.internals.ModelLoader
```

Comment:
pushed this change, to fix the build, which highlights one caveat: the DumpModel uses the ModelLoader, so we have to import from `internals`

we can do otherwise and export a function or something like that if you prefer, just bringing this to your attention

Reply:
lol there is also a codegen plugin tests that uses it haha

Reply:
Addressed in [f61403e](https://github.com/disneystreaming/smithy4s/pull/648/commits/f61403e42de2457cfa88fdd8c30b72e2b2c48b25)